api/v1alpha1/virtualmachineimage_types.go

```diff
@@ -73,9 +73,25 @@ type VirtualMachineImageStatus struct {
 	// Deprecated
 	PowerState    string `json:"powerState,omitempty"`
 
-	// SupportedGuestOS indicates whether the VirtualMachineImage's osType is supported
-	// by the hosts in the cluster and VM Service
-	SupportedGuestOS *bool  `json:"supportedGuestOS,omitempty"`
+	// ImageSupportedState indicates whether the VirtualMachineImage is supported by VM Service,
+	// A VirtualMachineImage is supported by VM Service if:
+	// - If OS Type is of Linux Family
+	// - If OS Type is supported by hosts in the cluster
+	// - If the image is a photon image or if the image is prepared by VI admin by adding vmware-system.vmsvc.compatibility key
```

Comment:
Let's not call out the explicit key here (if we have to, we will document it elsewhere), but rather simply state "The image has been specifically prepared for VM Service consumption"